recipes/sos/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "sos" %}
+{% set version = "0.9.16.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d460cf2e9822dffe5b6e9f11b87908b9e000149fd274ff39805473d555db2469" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [not py36]
```

Comment:
This could be a `noarch: python` I think.

Reply:
I agree.